.solhint.json

```diff
@@ -3,7 +3,7 @@
   "plugins": ["prettier"],
   "rules": {
     "code-complexity": ["error", 7],
-    "compiler-version": ["error", "^0.8.0"],
+    "compiler-version": ["error", "^0.7.3"],
```

Comment:
why not 0.7.4?

Reply:
The rest of the Inverse repos use 0.7.3. Will keep it at 0.7.3 for convenience.